lib/shoryuken/util.rb

```diff
@@ -4,13 +4,13 @@ def logger
       Shoryuken.logger
     end
 
-    def fire_event(event, reverse = false)
+    def fire_event(event, options = {}, *event_args)
```

Comment:
> Such that event options is meant to be hash?

@hasbrettn I was thinking on that for making the option more explicit, otherwise it would support `fire_event(..., false, queue_name, something_else)` and `fire_event(..., false, something_else, queue_name)`. If it becomes a hash, `queue_name` will be always `queue_name: `, no matter the order it was passed in. WDYT?